lib/elixir/lib/code.ex

```diff
@@ -800,6 +800,12 @@ defmodule Code do
   and the second one is its bytecode (as a binary). A `file` can be
   given as second argument which will be used for reporting warnings
   and errors.
+
+  **Warning**: `string` can be any Elixir code and some code can be executed
+  with the same privileges as the Elixir compiler: this means that such code
+  could compromise the machine (for example by executing system commands).
+  Don't use `oompile_string/2` with untrusted input (such as strings coming
```

Comment:
compile_string is misspelt. I would use commas and not parentheses.

Reply:
Also, it is not that "some code" can be executed with elixirc privileges. "All" the code is executed like that

Reply:
@eksperimental not all code is run during compilation, that was the case for this, not that some will be executed with `elixirc` privileges.

Reply:
Fixed wording.